code/game/objects/items/twohanded.dm

```diff
@@ -536,6 +537,13 @@
 				src.war_cry = input
 
 /obj/item/twohanded/spear/CheckParts(list/parts_list)
+	var/obj/item/shard/tip = locate() in parts_list
+	if (istype(tip, /obj/item/shard/plasma))
```

Comment:
Fix the materials (PLASMA), do ++ instead of literal numbers since the spear has its initial values to begin with.

Reply:
there is no guarantee that this proc will run only once

Reply:
OK I made a PR instead
https://github.com/vuonojenmustaturska/tgstation/pull/2